pcsx2/CDVD/CDVD.cpp

```diff
@@ -1048,9 +1048,9 @@ u8 cdvdRead(u8 key)
 			CDVD_LOG("cdvdRead07(Break) %x", 0);
 			return 0;
 
-		case 0x08:  // STATUS
-			CDVD_LOG("cdvdRead08(Status) %x", cdvd.Status);
-			return cdvd.Status;
```

Comment:
You're right. This looks very copypasta buggy with 2 cases both returning Status.